### ISSUE TITLE
Provide a way to export Solr Statistics to CSV for analysis via external tools

### ISSUE THREAD

**Issue by tdonohue** [DSpace/DSpace#8573]
**Is your feature request related to a problem? Please describe.**
This was brought up in a DCAT discussion today, and was supported by several DCAT members.

DCAT members noted they have users who'd like to be able to export usage statistics to Excel/CSV for easier manipulation in reports and/or analysis from external tools.  Presumably, this would include also exporting subsets of data (e.g. export a single Community's statistical data to CSV).

**Describe alternatives or workarounds you've considered**
Currently, a `./dspace stats-util -export` commandline tool exists which already allows for export to CSV (in order to migrate stats from one system to another).  Perhaps this could be updated, or enhanced to better support exporting stats to CSV?

https://wiki.lyrasis.org/display/DSDOC7x/SOLR+Statistics+Maintenance#SOLRStatisticsMaintenance-ExportSOLRrecordstointermediateformatforimportintoanothertool/instance

**Comment by mwoodiupui** [comment]
Exporting cases in a form that can be ingested by Real Statistical Tools:  couldn't agree more.

While we are looking at this, I would like to see some discussion of why we use Solr to store cases.

**Comment by mwoodiupui** [comment]
We still need a way to get these data into other tools for arbitrary analyses.

**Comment by tdonohue** [comment]
@mwoodiupui : Good point, yes this ticket is still "wanted" even though it hasn't had activity in >3 years.

In fact, I think this overlaps now with the "Enhanced Statistics ( and exporting statistical reports)" feature coming in DSpace-CRIS, as I know that DSpace-CRIS has the ability to export statistical reports to CSV (and also export statistical charts to images).  See `#25` in the [feature differences list](https://wiki.lyrasis.org/display/DSPACE/DSpace-CRIS+and+DSpace+Differences).

So, I'm going to remove the `stale` label & flag this as "blocked by" #11782, as I believe this feature may be coming in that ticket.

(This is also related to #11853)
